cetmix_tower_server/models/cx_tower_server_template.py

```diff
@@ -299,76 +311,136 @@ def _prepare_server_values(self, **kwargs):
 
                     values[field] = new_records
 
-            # custom specific variable values
+            # Handle configuration variables if provided.
             configuration_variables = kwargs.pop("configuration_variables", None)
-            line_ids_variables = kwargs.pop("line_ids_variables", None)
+            configuration_variable_options = kwargs.pop(
+                "configuration_variable_options", {}
```

Comment:
I don't see this one added to the docstring similar to how `configuration_variables` is, is it missed?